### PR TITLE
Add additional readme documentation.

### LINKED ISSUE #199: Add README to context-propagation-micrometer and relocation modules
Could you please add a README to the Maven modules [`context-propagation-micrometer`](https://github.com/talsma-ict/context-propagation/tree/develop/context-propagation-micrometer) and [`relocation`](https://github.com/talsma-ict/context-propagation/tree/develop/relocation)?

This would make it easier to understand what these modules are for.

### PR DESCRIPTION
**Pull request by sjoerdtalsma**
Documented the _relocation_ and _micrometer_ modules.

This fixes #199.

### CODE REVIEW

**Comment by sjoerdtalsma** [comment]
@Marcono1234 could you please have a look whether this is what you had in mind?

**Review by Marcono1234 — COMMENTED** [review]
Thank you very much! Yes that looks pretty good. I have left some minor review comments mostly regarding capitalization.

Should the Micrometer module also be mentioned in the root README?

File: relocation/readme.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Relocation
+
+This directory contains [maven relocation](https://maven.apache.org/guides/mini/guide-relocation.html) poms
+for modules that were once published in maven central, 
+but whose _artifactId_ have either been renamed or moved to another _groupId_.
```
[the remark this answers is not in the document]
Comment (sjoerdtalsma) [reply relocation/readme.md:5]: for module**s** that (...) _have_ either been renamed or (...)

I think with the plural modules being the subject, this should remain _have_ ?

[the remark this answers is not in the document]
Comment (Marcono1234) [reply relocation/readme.md:5]: Yes, you are right I missed that. But here "have" / "has" also refers to the (singular) artifactId. Maybe the following would be better then:
```suggestion
but whose _artifactId_ has either been renamed or which have been moved to another _groupId_.
```

**Pull request merged**